cfme/containers/provider/openshift.py

```diff
@@ -20,10 +20,15 @@ class OpenshiftProvider(ContainersProvider):
     db_types = ["Openshift::ContainerManager"]
 
     def __init__(self, name=None, credentials=None, key=None,
-                 zone=None, hostname=None, port=None, provider_data=None, appliance=None):
+                 zone=None, hostname=None, port=None, provider_data=None, appliance=None,
+                 sec_protocol=None):
         super(OpenshiftProvider, self).__init__(
             name=name, credentials=credentials, key=key, zone=zone, hostname=hostname, port=port,
-            provider_data=provider_data, appliance=appliance)
+            provider_data=provider_data, appliance=appliance, sec_protocol=sec_protocol)
+
+        # Here drop a selector of OSE versions?
+        if self.appliance.version.vstring > '5.8.0.3':
+            self.sec_protocol = sec_protocol
```

Comment:
You don't need the version check here. You can just do `self.sec_protocol = sec_protocol`, however, the version check should be within `_form_mapping`. Also, is this specific to OpenshiftProvider? If not, it should be in ContainersProvider.